### PR TITLE
colorful logging with winston logger

### PR DESCRIPTION
**Pull request by farhancdr**
Created a simple logger with winston. 
It will be logging to console like this...

loggerScreenShot

It won't break any other codes. Now we can change all `console.log` to `logger.info` or `logger.warn` or `logger.error` .

### CODE REVIEW

File: server/src/core/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 
 const setup = async () => {
-    const logger = require('pino')()
+    const logger = require('./logger');
```
Comment (foyzulkarim) [inline server/src/core/index.js:4]: Thanks a lot for your PR. here is a suggestion. 
please do a conditional logger selection.
eg. 
```
const logOption = process.env.logger;
if(logOption==='pino') return createPino() 
return createWinston();
```

**Review by foyzulkarim — CHANGES_REQUESTED** [review]
Please do a conditional change.  :) 
Again thanks

File: server/src/core/index.js (hunk 1 above)
Comment (farhancdr) [reply server/src/core/index.js:4]: added the conditional logger selection on `logger.js` file. Kindly check that.

**Comment by farhancdr** [comment]
@foyzulkarim vaiya, I've added the conditional logger selection functionality on `logger.js` file. Kindly check that.

File: server/src/core/index.js (hunk 1 above)
Comment (foyzulkarim) [reply server/src/core/index.js:4]: thanks. :)

**Review by foyzulkarim — APPROVED** [review]
Looks good. Thanks.

**Pull request merged**